.github/workflows/build.yml

```diff
@@ -7,10 +7,16 @@ jobs:
     runs-on: ubuntu-20.04
     steps: 
       - uses: actions/checkout@v1
-      - name: Build bootloader and partition table
+      - name: Build bootloader and partition table with DIN
         uses: docker://docker.io/espressif/idf:release-v4.3
         with:
-          args: ./build.sh
+          args: ./build.sh -m DIN
```

Comment:
Did you mean "DIO" (which is the default)?

By the way, i'd recommend running build.sh twice in the same job; otherwise espressif/idf image will be downloaded twice.

Reply:
Yes, I meant DIO, sorry for the typo.

> By the way, i'd recommend running build.sh twice in the same job; otherwise espressif/idf image will be downloaded twice.

I'll change this.